tests/hash_unittest.py

```diff
@@ -62,6 +70,11 @@ def test_functools_partial(self):
             ),
         )
 
+    def test_bound_method(self):
+        first_obj = MyFoo("First")
+        second_obj = MyFoo("Second")
+        self.assertNotEqual(sis_hash_helper(first_obj.get_data), sis_hash_helper(second_obj.get_data))
```

Comment:
could you please also add a test like
```
self.assertEqual(sis_hash_helper(first_obj.get_data), b"correct value goes here")
```
so we test that the value is unique and constant?